packages/components/src/toolbar/stories/index.js

```diff
@@ -30,10 +54,10 @@ function InlineImageIcon() {
 	);
 }
 
-/* eslint-disable no-restricted-syntax */
-export const _default = () => {
+export const Default = () => {
```

Comment:
This isn't super important because `Toolbar` itself doesn't have a lot of props anyway, but I think a general pattern we can use to keep the main component "controls-ready" is to pass in all the content via the `children` arg.

```jsx
const Template = ( props ) => <Toolbar { ...props } />;

export const Foo = Template.bind( {} );
Foo.args = {
	label: 'Options',
	children: <ToolbarGroup>{/* whatever */}</ToolbarGroup>,
};

Reply:
Good point, I refactored the stories to use `Template.bind()` + `args` syntax as suggested